backend/cmd/server/main.go

```diff
@@ -288,9 +254,8 @@ func gracefulShutdown(logger *log.Logger, server *http.Server) {
 	}
 
 	// Shutdown observability service
-	observabilitySvc := observability.GetService()
-	if observabilitySvc != nil {
-		observabilitySvc.Shutdown()
+	if ObservabilitySvc != nil {
```

Comment:
without accessing ObservabilitySvc in main.go, shall we create an unregister function in servicemanager and have this logic there?